devices/Ssd13xx/Ssd1306.cs

```diff
@@ -28,17 +33,42 @@ public Ssd1306(I2cDevice i2cDevice) : base(i2cDevice)
         {
         }
 
+        /// <summary>
+        /// Initializes new instance of Ssd1306 device that will communicate using I2C bus.
+        /// A single-chip CMOS OLED/PLED driver with controller for organic/polymer
+        /// light emitting diode dot-matrix graphic display system.
+        /// </summary>
+        /// <param name="i2cDevice">The I2C device used for communication.</param>
+        /// <param name="rstPinNumber">Reset pin (some displays might be wired to share the microcontroller's
+        /// reset pin).</param>
+        public Ssd1306(I2cDevice i2cDevice, int rstPinNumber) : base(i2cDevice, rstPinNumber)
```

Comment:
If you pass a pin number, you need to allow the possibility to pass the GPIO controller and also add the shouldDispose boolean to dispose it or not at the end. You'll find the pattern in other bindings.